int/hwsc-file-transaction-svc/hwsc-file-transaction-svc.js

```diff
@@ -24,50 +24,67 @@ function getStatus(callback) {
   const request = {};
   client.getStatus(request, (err, response) => {
     if (!err) {
-      grpc.closeClient(client);
-    }
-    callback(err, response);
-  });
+    grpc.closeClient(client);
+  }
+  callback(err, response);
+});
 }
 
-function uploadFile(filePath, fileName, callback) {
+function createUserFolder(uuid, callback) {
   if (typeof callback !== 'function') {
     console.error('callback not a function');
     return;
   }
+  const id = uuid;
+  console.log(id);
+  const request = {};
+  request.uuid = uuid;
+  const server = client.createUserFolder(request, (err, response) => {
```

Comment:
I'm confused by this line, why is this function stored in a variable? When/how do you use it?

Reply:
modify this to 
```
client.createUserFolder(request, (err, response) => {
    if (!err) {
      grpc.closeClient(client);
    }
    callback(err, response);
  });
}
```